core/commands/storage.go

```diff
@@ -212,6 +210,9 @@ Receive proofs as collateral evidence after selected nodes agree to store the fi
 		if found && price < HostPriceLowBoundary {
 			return fmt.Errorf("price is smaller than minimum setting price")
 		}
+		if found && price >= math.MaxInt64 {
+			return fmt.Errorf("price should be  smaller than max int64")
+		}
 		if !found {
 			// todo: leave it for either no host prices or no user price, can be deleted later
 			price = HostPriceLowBoundary
```

Comment:
Can you update this to set it to the default hub price? Otherwise host won't pass the check...